justice/util.py

```diff
@@ -0,0 +1,29 @@
+"""Utility functions"""
+
+from collections import namedtuple
+import numpy as np
+
+LC = namedtuple('LC', ('x', 'y', 'yerr'))# time, flux/mag, flux/mag err
+
+Aff = namedtuple('Aff', ('tx', 'ty', 'dx', 'dy'))# translation, dilation in x, y
+
+def make_aff(list):
```

Comment:
can you just call Aff() instead of doing this?

Reply:
I did that because I thought it looked sloppy to keep writing `Aff(params[0], params[1], params[2], params[3])`, but I don't know of a way to get canned optimizers to handle namedtuples rather than lists and arrays.